src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -35,32 +37,32 @@ import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {ProfileContact} from '../../components/Items';
 import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
 import {getOfficialAnonUsername} from '../../utils/string/StringUtils';
+import useGroupInfo from '../GroupInfo/hooks/useGroupInfo';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useGroupInfo` hook has been introduced. It's a good addition as it provides functionalities related to group information like leaving a group or reporting a group.
```diff
+ import useGroupInfo from '../GroupInfo/hooks/useGroupInfo';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->